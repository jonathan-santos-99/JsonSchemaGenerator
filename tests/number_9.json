{ "latitude": -64.791975 }