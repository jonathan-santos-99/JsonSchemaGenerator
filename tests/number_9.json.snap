0
eyIkc2NoZW1hIjoiaHR0cDovL2pzb24tc2NoZW1hLm9yZy9kcmFmdC0wNC9zY2hlbWEjIiwidHlwZSI6Im9iamVjdCIsInByb3BlcnRpZXMiOnsibGF0aXR1ZGUiOnsidHlwZSI6Im51bWJlciJ9fSwicmVxdWlyZWQiOlsibGF0aXR1ZGUiXX0=
